src/components/Header/index.jsx

```diff
@@ -1,102 +1,120 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
nit: do we have normal standards related to 1 component per file? If so, please file ticket. if that's not a thing we do, ignore me.

Reply:
We don't have a strict standard, usually just a judgement call based on how complex/large the components are. E.g., Paragon has components that utilize sub-components, but not all those sub-components have their own file.

Reply:
my take: it is okay to have multi component in a single file if they are always expected to be used together and are very close in what they do in the ui. Generally I would go for one file per component, but yeah it's a bit subjective when this kind of grouping is better, in this case it seems like it is a fine choice given how small and closely knit together they are